index.html

```diff
@@ -3,7 +3,487 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <link rel="stylesheet" href="css/styles.css" />
+    <title>Yotube</title>
   </head>
-  <body></body>
+  <body>
+    <nav>
+      <div class="general">
+        <div class="menuIcon">
+          <section class="menuIcon">
+            <img
+              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAAV1BMVEX///90d3lxdHZtcHJqbW/AwcHk5eWVmJlnam1rbnCpq6yBhIb5+fnHyMng4eH29veKjI6ztLWipKV5fH6OkZLZ2tvq6+vLzMy5u7zIycnR0tLCw8OanJ7n5uYQAAACyUlEQVR4nO3di27iMBCFYcfOFodbEiBAgfd/ziXQSlvJNqkWCc3h/54gR3ac1p4xzgEAAAAAAAAAAAAAAAAoa4fVH2tWQzs13vK4qJtgT1MvjsspAfsYfGWTD7F/mG+/Da9+zv8StvtywKG2On7ffD2UAn7EVz/gE8SPfMCl7Rn6LeTXm5n1KXrnZ7mAa4U5Oopr7SHMD2KrMoTXQUz/eXPQWGdG4ZBMOFeZpNdpOk8m3Akl3CUTLoQSLt40YSeUsEsm3Agl3CQTnoW+Fudkwr3QFz/zT6LMUpNZaJw71a9+tCepT5mEKoOYHUKZNzH3Fo7OChFjeiH9crQfMR5LAZ37jLbfRR8/ywGv72JneEPR192D7dKbYReb4O0JTdwV90r/0Z76zcyaTX+afDQDAAAAAAAAAAAAYDL180P1M2D5c/yV+VqMVTmgfD2NfE2UROtasXlNvjbxIFNfmm6ZkRnC/CCKVJeOMhWm+rX6+v0W+j0zMgvNG3d26fcf6veQ6vcB6/dy6/fjv8GdCiqDmL8X4w3uNtH4J790P41zg/GttnGz7cGeqfw9UU7/ri93u68tmryvLU68r23UDquzNb+4cw8AAAAAAAAAAADAdKd5V726i+nXqm6evSD5p2UfGm/xnNv7JvQTDoFXwXKpQggP2p6cu1ivqImXcsCd5QG8C+nq2S8b+wGvEdOF+jcn61P0LubXVIsraIrPBTwqzNFRyDVZvvrBnigdcK3SnFdVdbpIWGaSZqfpRWWhuS416c8+nV2GvG1nl0gvwijTj9ALraXpYuihefWDPU2TqdcX+uKnA+o0IGbaD51rVQaxzta0i6w1mXXmRuKTWPhRMueWlf2IviruKLbG257GxqdHjSUX091r/tFm4mhYRJNb3uOmd1xM+0mr9Xzb1PY023mxt/Kn5f7Dmv3kvjUAAAAAAAAAAAAAUPUXCyx+ltjBidoAAAAASUVORK5CYII="
+              alt="menu icon"
+            />
+          </section>
+        </div>
+        <div class="youtubeIcon">
+          <img src="./images/YoutubeIcon.PNG" alt="youtube icon" />
+        </div>
+        <div class="searchInput">
+          <div class="inputSection">
+            <input type="text" placeholder="Search" />
+          </div>
+          <div class="icon">
+            <img src="./images/searchIcon.PNG" alt="Search" />
+          </div>
+        </div>
+        <div class="rightSection">
+          <section class="webCam">
+            <img src="./images/Webcam_Youtube.PNG" alt="webcam icon"  />
+          </section>
+          <section class="dashboard">
+            <img src="./images/dashboard_youtube.PNG" alt="dashboard icon" />
+          </section>
+          <section class="notification">
+            <img src="./images/notification_youtube.PNG" alt="notification icon"  />
+          </section>
+          <section class="profile">
+            <img src="./images/Profile.jpg" alt="profile icon"  />
+          </section>
+        </div>
+      </div>
+    </nav>
+    <div class="container">
+      <div class="leftSection">
```

Comment:
Let us improve semantics by using ```<main>``` instead of ```<div class="leftSection">```

Reply:
Replaced `<div class="leftSection">` with `<main>`